cache/binary/tags.go

```diff
@@ -124,6 +124,13 @@ func init() {
 	addCommonKey("addr:postcode")
 	addCommonKey("addr:housenumber")
 
+	addCommonKey("osm_changeset")
```

Comment:
Arg... that should have been `_changeset_`, `_version_`, etc. (markdown made it italic).
Your osm2pgsql stuff is expecting an osm_changeset column? You could configure that:

                {
                    "type": "integer",
                    "name": "osm_changeset",
                    "key": "_changeset_"
                },